openjdk.test.load/config/inventories/mauve/mauve_multiThread_exclude.xml

```diff
@@ -1,4 +1,15 @@
 <inventory>
+	<!-- 
+	Tests that use SecurityManager fail intermittently when run in a 
+	multi-threaded environment due to timing issues. 
+	Issue reference: https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/271
+	 -->
+	<mauve class="gnu.testlet.java.security.ProtectionDomain.Implies"/>
+	<mauve class="gnu.testlet.java.net.URLConnection.getRequestProperties"/>
+	<mauve class="gnu.testlet.java.net.URLConnection.getPermission"/>
+	<mauve class="gnu.testlet.java.net.URLStreamHandler.Except"/>
```

Comment:
Should tests that make a call to url.openConnection() from running in Multi-threaded Mauve load be excluded? @pshipton